pkg/daemon/daemon.go

```diff
@@ -632,7 +632,6 @@ func (dn *Daemon) Run(stopCh <-chan struct{}, exitCh <-chan error) error {
 	case err := <-exitCh:
 		// This channel gets errors from auxiliary goroutines like loginmonitor and kubehealth
 		glog.Warningf("Got an error from auxiliary tools: %v", err)
-		return err
```

Comment:
can we go back to infinite `for` loop instead of using `select` so that this function keeps running when something on exitCh is sent?

Reply:
Rather than an infinite loop, I pushed a change with a separate Go routine that watches the channel and logs instead of returning the error. `exitCh` should really be called `errorCh` since it bubbles errors but is not used to "exit".